modules/qt/hm.nix

```diff
@@ -86,11 +94,19 @@
             ''
               [Appearance]
             ''
-            + lib.optionalString (config.qt.style ? name) ''
+            + lib.optionalString (config.qt.style.name != null) ''
```

Comment:
We want code to be as resilient as possible to upstream changes, not as minimal as possible. This trades the former for no seeming improvement.

Reply:
My bad, the point wasn't to be minimal or something like that. I honestly thought this was just to make sure stylix doesn't attempt to write the value if it is null, since it defaults to that. I'll revert it soon